test/EFCore.PG.FunctionalTests/F1NpgsqlFixture.cs

```diff
@@ -1,12 +1,21 @@
 ﻿using Microsoft.EntityFrameworkCore;
 using Microsoft.EntityFrameworkCore.TestModels.ConcurrencyModel;
 using Microsoft.EntityFrameworkCore.TestUtilities;
-using Npgsql.EntityFrameworkCore.PostgreSQL.Metadata.Conventions;
 using Npgsql.EntityFrameworkCore.PostgreSQL.TestUtilities;
 
 namespace Npgsql.EntityFrameworkCore.PostgreSQL
 {
-    public class F1NpgsqlFixture : F1RelationalFixture
+    public class F1UIntNpgsqlFixture : F1NpgsqlFixtureBase<uint?>
```

Comment:
F1? Definitely  some help required here to understand the sacral meaning of that prefix.

Reply:
Formula 1 if I'm not mistaken :) Take a look at the entity types in F1Context...